### PR TITLE
(asm) Update doc for protection navigation release

### PR DESCRIPTION
**Pull request by Moutix**
### What does this PR do? What is the motivation?

Following the release of the new navigation we want to update the different links we have in ASM threats

Configuration for In App Waf will now be in ASM -> Protection main menu

### Merge instructions

- [ ] Please merge after release of the new navigation (ETA this week)
- [ ] Please merge after reviewing

### CODE REVIEW

File: content/en/security/application_security/threats/protection.md [hunk 1]
```diff
@@ -77,11 +77,9 @@ View blocked security traces in the [Trace Explorer][11] by filtering on the fac
...
+2. **Associate your ASM/Remote Configuration-enabled services with a policy**. After Remote Configuration is enabled on a service, navigate to **Security > Protection > Application > [In-App WAF][9]**. The service will appears under the _Datadog Monitoring-only_ policy by default. Datadog Monitoring-only is a managed policy and is read-only, meaning you cannot modify the status (monitoring, blocking, or disabled) for individual rules.
 
-   If you need granular control, clone the _Datadog Recommended_ policy to create a custom policy where rule statuses can be modified. Associate one or more of your services with this custom policy.
-
-3. **Configure blocking for each service**. By default, no requests are blocked even if certain In-App WAF rule statuses are blocking. Navigate to **Security > Configuration > Application > [In-App WAF][13]** to turn on blocking mode for a service. See [In-app WAF rules][19] for more information.
+   If you need granular control, clone one of the available policy to create a custom policy where rule statuses can be modified. Associate one or more of your services with this custom policy.
```
Comment (M4TTRX) [inline content/en/security/application_security/threats/protection.md:82]: This doc mentions nothing of default policies, maybe we could add a sentence?

Suggestion:
_If you would like to change the policy applied by default to your services, you can update your default policy. From the In-App-WAF, click on the policy you would like to set as default, then click on  "Actions" > "Set this policy as default"._

**Review by M4TTRX — APPROVED** [review]

File: content/en/security/application_security/threats/protection.md (hunk 1 above)
Comment (Moutix) [reply content/en/security/application_security/threats/protection.md:82]: good point!

File: content/en/security/application_security/threats/protection.md [hunk 2]
```diff
@@ -77,11 +77,11 @@ View blocked security traces in the [Trace Explorer][11] by filtering on the fac
...
 
-2. **Associate your ASM/Remote Configuration-enabled services with a policy**. After Remote Configuration is enabled on a service, it appears under the _Datadog Recommended_ policy by default. Datadog Recommended is a managed policy and is read-only, meaning you cannot modify the status (monitoring, blocking, or disabled) for individual rules.
+2. **Associate your ASM/Remote Configuration-enabled services with a policy**. After Remote Configuration is enabled on a service, navigate to **Security > Protection > Application > [In-App WAF][9]**. The service will appears under the _Datadog Monitoring-only_ policy by default. Datadog Monitoring-only is a managed policy and is read-only, meaning you cannot modify the status (monitoring, blocking, or disabled) for individual rules.
 
-   If you need granular control, clone the _Datadog Recommended_ policy to create a custom policy where rule statuses can be modified. Associate one or more of your services with this custom policy.
+   If you need granular control, clone one of the available policy to create a custom policy where rule statuses can be modified. Associate one or more of your services with this custom policy.
```
Comment (drichards-87) [inline content/en/security/application_security/threats/protection.md:82]:
```suggestion
   If you need granular control, clone one of the available policies to create a custom policy where rule statuses can be modified. Associate one or more of your services with this custom policy.
```

File: content/en/security/application_security/threats/protection.md [hunk 3]
```diff
@@ -77,11 +77,11 @@ View blocked security traces in the [Trace Explorer][11] by filtering on the fac
...
 
-3. **Configure blocking for each service**. By default, no requests are blocked even if certain In-App WAF rule statuses are blocking. Navigate to **Security > Configuration > Application > [In-App WAF][13]** to turn on blocking mode for a service. See [In-app WAF rules][19] for more information.
+   If you would like to change the policy applied by default to your services, you can update your default policy. From the In-App-WAF, click on the policy you would like to set as default, then click on "Actions" > "Set this policy as default".
```
Comment (drichards-87) [inline content/en/security/application_security/threats/protection.md:84]:
```suggestion
   To change the policy applied by default to your services, you can update your default policy. From the In-App-WAF, click the policy you would like to set as default, then click **Actions** > **Set this policy as default**.
```

**Comment by Moutix** [comment]
Thks for the suggestion! I've added them

**Review by drichards-87 — APPROVED** [review]
Looks good! Let me know when this is ready to be merged. You can also ping the team in Slack and someone will merge it for you.

**Comment by Moutix** [comment]
@drichards-87 @aliciascott We just released the new navigation, so we can merge this PR and this one as well! https://github.com/DataDog/documentation/pull/21218

**Pull request merged**